updatecli/updatecli.d/letsencrypt_cert.ci.yaml

```diff
@@ -56,6 +56,6 @@ actions:
     kind: github/pullrequest
     scmid: default
     spec:
-      title: Generate a new password for the Azure Service Principal "cert.ci.jenkins.io - Let's Encrypt" with (https://github.com/jenkins-infra/jenkins-infra/blob/d76e72a6c1ee06a0d321ed4d04c5d3e1b81d061d/hieradata/clients/controller.cert.ci.jenkins.io.yaml#L27)
+      title: Generate a new password for the Azure Service Principal "cert.ci.jenkins.io - Let's Encrypt". Set the new password value in the following encrypted file: <https://github.com/jenkins-infra/jenkins-infra/blob/d76e72a6c1ee06a0d321ed4d04c5d3e1b81d061d/hieradata/clients/controller.cert.ci.jenkins.io.yaml#L27> (open a PR : https://github.com/jenkins-infra/jenkins-infra/edit/production/hieradata/clients/controller.cert.ci.jenkins.io.yaml).
```

Comment:
```suggestion
      title: Generate a new password for the Azure Service Principal "cert.ci.jenkins.io - Let's Encrypt"
      description: Set the new password value at `sp_client_secret` in the `profile::letsencrypt::dns_azure:` section of the encrypted file: https://github.com/jenkins-infra/jenkins-infra/blob/production/hieradata/clients/controller.cert.ci.jenkins.io.yaml.
```
Add description, no "absolute" ref, and removing the "open a PR link", a bit redundant.